.travis.yml

```diff
@@ -17,7 +18,7 @@ matrix:
         # https://github.com/travis-ci/travis-ci/issues/9368#issuecomment-395354755
         - rm "${JAVA_HOME}/lib/security/cacerts"
         - ln -s /etc/ssl/certs/java/cacerts "${JAVA_HOME}/lib/security/cacerts"
-    - jdk: openjdk11
```

Comment:
Because openjdk11 wasn't listed above, this effectively removes openjdk11 from the matrix!

Reply:
Thanks for the catch!